pkg/apimachinery/identity/context_test.go

```diff
@@ -17,126 +17,10 @@ func TestRequesterFromContext(t *testing.T) {
 	})
 
 	t.Run("should return user set by ContextWithUser", func(t *testing.T) {
-		expected := &dummyUser{UID: "AAA"}
+		expected := &identity.StaticRequester{UserUID: "AAA"}
```

Comment:
kind of weird having the test files in a separate package but the same directory, took me a minute to realize why we needed `identity.` here

Reply:
ya -- that is the difference of `{package}_test` or just `{package}` -- it lets you add tests that use the public interfaces rather than internal ones.